cardano-tracer/src/Cardano/Tracer/Acceptors/Client.hs

```diff
@@ -89,6 +89,7 @@ doConnectToForwarder
 doConnectToForwarder snocket address netMagic timeLimits app =
   connectToNode
     snocket
+    mempty -- LocalSocket does not require to be configured
```

Comment:
Why not?

Reply:
@angerman might know more about this

Reply:
Ok, we need to leave a comment beyond "LocalSocket does not require to be configured"

Reply:
There's no need to configure unix sockets or named pipes on Windows.

Reply:
@coot provided the following explanation:
> local sockets (e.g. unix sockets or named pipes on Windows), do not need any configuration; we only need to configure node-to-node sockets. 

@Jimbo4350 is that sufficient if we leave it as a comment?

Reply:
a race condition :)

Reply:
I changed the comment to the one that @angerman provided above

Reply:
> There's no need to configure unix sockets or named pipes on Windows.

Why?

Reply:
We never did anything special to them.  Adding `SO_REUSEADDR` is probably not that useful as one can simply delete the file, something one cannot do when using IPv4/v6.